tests/NHSD.BuyingCatalogue.API.IntegrationTests/Features/Preview/Display Marketing Page Preview Solution Section.3507.feature

```diff
@@ -0,0 +1,43 @@
+Feature: Display Marketing Page Preview Solution Section
+	As a Supplier
+    I want to manage Marketing Page Information for the Solution Section
+    So that I can ensure the information is correct
+
+Background:
+    Given Suppliers exist
+        | Id    | SupplierName |
+        | Sup 1 | Supplier 1   |
+        | Sup 2 | Supplier 2   |
+    And Solutions exist
+        | SolutionID | SolutionName   | LastUpdated | SupplierStatusId | SupplierId |
+        | Sln1       | MedicOnline    | 19/11/2019  | 1                | Sup 1      |
+        | Sln2       | TakeTheRedPill | 15/11/2019  | 1                | Sup 2      |
+        | Sln3       | PracticeMgr    | 20/11/2019  | 1                | Sup 2      |
+    And Framework Solutions exist
+        | SolutionId | IsFoundation |
+        | Sln1       | true         |
+        | Sln2       | false        |
+
+@3507
+Scenario: 1. Solution section is presented where the solution isFoundation is true
+    When a GET request is made for solution preview Sln1
+    Then a successful response is returned
+    And the string value of element supplierName is Supplier 1
+    And the string value of element isFoundation is True
+    And the last updated date in the solution is 19/11/2019
+
+@3507
+Scenario: 2. Solution section is presented where the solution isFoundation is false
+    When a GET request is made for solution preview Sln2
+    Then a successful response is returned
+    And the string value of element supplierName is Supplier 2
+    And the string value of element isFoundation is False
+    And the last updated date in the solution is 15/11/2019
+
+@3507
+Scenario: 3. Solution is not linked to Framwork Solution
```

Comment:
Framework?

Reply:
This is meant to be Foundation, I assume? these were a carbon copy of the Public tests, as they weren't being tested in Preview. I'll make the change in both!